README.md

```diff
@@ -82,6 +82,7 @@ Rule                              | Recommended      | Options
 [no-unsafe-spy][]                 | 1                |
 [valid-expect][]                  | 1                |
 [prefer-jasmine-matcher][]        | 1                |
+[prefer-toHaveBeenCalledWith][]   | 0                |
```

Comment:
that's a good point, I think the best compromise is to make this rule not warn for `not.toHaveBeenCalled()`